src/block-1.0.ts

```diff
@@ -11,75 +11,11 @@
 // under the License.
 
 import { Rewarded } from "../generated/PoS-1.0/PoS"
-import { BlockProduced } from "../generated/BlockSelector-1.0/BlockSelector"
-import { Block } from "../generated/schema"
-import * as chains from "./chain"
-import * as nodes from "./node"
-import * as summary from "./summary"
-import * as users from "./user"
+import { handleRewardedInner, handleBlockProduced } from "./block-common"
 
 export function handleRewarded(event: Rewarded): void {
     let reward = event.params.userReward.plus(event.params.beneficiaryReward)
-
```

Comment:
Refacto: similar code from `block.ts` and `block-1.ts` have been moved to `block-common.ts`